kairon/shared/pyscript/callback_pyscript_utils.py

```
@@ -0,0 +1,333 @@
+    @staticmethod
+    def save_as_pdf(text: str, bot: str, sender_id:str):
+        try:
+            _, media_id = UserMedia.save_markdown_as_pdf(
+                bot=bot,
+                sender_id=sender_id,
+                text=text,
+                filepath="report.pdf"
+            )
+            return media_id
+        except Exception as e:
+            raise Exception(f"encryption failed-{str(e)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)